bigquery/bigquery_quickstart/main.go

```diff
@@ -30,11 +30,8 @@ func main() {
 	// Sets the name for the new dataset.
 	datasetName := "my_new_dataset"
 
-	// Creates a Dataset instance.
-	dataset := client.Dataset(datasetName)
```

Comment:
note: removed this. what do you think?

the comment was quite poorly worded. can change it back and remove the comment?